packages/big-design/src/components/Worksheet/editors/SelectEditor/styled.ts

```diff
@@ -10,17 +10,21 @@ export const SelectWrapper = styled.div<{ isEdited: boolean }>`
     &:hover:not([disabled]) {
       border: none;
     }
+
+    &[disabled] {
+      background-color: ${({ theme }) => theme.colors.white};
+    }
   }
 
   input {
     font-size: ${({ theme }) => theme.typography.fontSize.small};
     font-weight: ${({ theme }) => theme.typography.fontWeight.regular};
 
-    ${({ isEdited }) =>
-      isEdited &&
-      css`
-        background-color: ${({ theme }) => theme.colors.warning10};
-      `}
```

Comment:
Curious about this deletion 🤔

Reply:
+1 looks unrelated?

Reply:
Good catch, I think this was deleted by accident, let me 👀

Reply:
Turns out it is no longer needed since I changed the parent `span` to `transparent` and the input `inherits` the background. I will be, however, updating the `spam` to inherit too so that the `span` and `input` always have the same color of the `td`, in case we add extra color types in the future.